.env.example

```
@@ -22,3 +22,10 @@ WA_ACCESS_TOKEN=""
 WA_TEMPLATE_NAME="payment_failed_recover"
 WA_TEMPLATE_LANG="en_US"
 WA_GRAPH_API_URL="https://graph.facebook.com/v21.0"
+
+# HITL review queue auth (issue #34): the /hitl/* API is the one route group
+# behind x-api-key. Set the same demo value on both sides:
+#   GRABIT_API_KEY="demo-key"         -> server (apps/api)
+#   VITE_GRABIT_API_KEY="demo-key"    -> browser (apps/web/.env.local)
+GRABIT_API_KEY="demo-key"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)